.github/workflows/sdk.yml

```diff
@@ -7,11 +7,13 @@ on:
     branches:
       - 'main'
 
+permissions:
+  id-token: write
```

Comment:
This appears consistent with the docs at https://docs.npmjs.com/trusted-publishers#github-actions-configuration although there is some implicit magic going on behind the scenes that I don't quite grasp. But I don't need to if this works.